rules/macos/execution_initial_access_suspicious_browser_childproc.toml

```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2020/12/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of a suspicious browser child process. Adversaries may gain access to a system through a user
+visiting a website over the normal course of browsing. With this technique, the user's web browser is typically targeted
+for exploitation.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Browser Child Process"
+references = [
+    "https://objective-see.com/blog/blog_0x43.html",
+    "https://fr.slideshare.net/codeblue_jp/cb19-recent-apt-attack-on-crypto-exchange-employees-by-heungsoo-kang",
+]
+risk_score = 73
+rule_id = "080bc66a-5d56-4d1f-8071-817671716db9"
+severity = "high"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Initial Access", "Execution"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.parent.name:("Google Chrome", "Google Chrome Helper*", "firefox", "Opera", "Safari", "com.apple.WebKit.WebContent", "Microsoft Edge") and
+ process.name:("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget", "python*", "perl*", "php*", "osascript", "pwsh") and 
+ process.command_line != null and not 
+ process.args:( 
+                "/Library/Application Support/Microsoft/MAU*/Microsoft AutoUpdate.app/Contents/MacOS/msupdate", 
+                "hw.model", "IOPlatformExpertDevice", "/Volumes/Google Chrome/Google Chrome.app/Contents/Frameworks/*/Resources/install.sh",
+                "--defaults-torrc", "Chrome.app", "Framework.framework/Versions/*/Resources/keystone_promote_preflight.sh", 
+                "/Users/*/Library/Application Support/Google/Chrome/recovery/*/ChromeRecovery", "$DISPLAY", "GIO_LAUNCHED_DESKTOP_FILE_PID=$$"
+                )
```

Comment:
```suggestion
process where event.type in ("start", "process_started") and
  process.parent.name : ("Google Chrome", "Google Chrome Helper*", "firefox", "Opera", "Safari", "com.apple.WebKit.WebContent", "Microsoft Edge") and
  process.name : ("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget", "python*", "perl*", "php*", "osascript", "pwsh") and 
  process.command_line != null and 
  not process.args : 
    ( 
      "/Library/Application Support/Microsoft/MAU*/Microsoft AutoUpdate.app/Contents/MacOS/msupdate", 
      "hw.model", 
      "IOPlatformExpertDevice", 
      "/Volumes/Google Chrome/Google Chrome.app/Contents/Frameworks/*/Resources/install.sh",
      "--defaults-torrc", 
      "Chrome.app", 
      "Framework.framework/Versions/*/Resources/keystone_promote_preflight.sh", 
      "/Users/*/Library/Application Support/Google/Chrome/recovery/*/ChromeRecovery", 
      "$DISPLAY", 
      "GIO_LAUNCHED_DESKTOP_FILE_PID=$$"
    )
```

Reply:
could split out `parent.process.name` and `process.name` on new lines as well but it was getting a little too long when i did that